services/search/pkg/engine/bleve.go

```diff
@@ -6,6 +6,7 @@ import (
 	"math"
 	"path"
 	"path/filepath"
+	"reflect"
```

Comment:
Any chance we can achieve this without `reflect`? We are trying to avoid using reflect on runtime

Reply:
I asked @butonic and @micbar and they said it was ok to use this. Any json (un)marshalling which has no generated (un)marshal functions uses reflection, I'm not so sure this is an actual problem.

Speaking of that: this can be achieved without `reflect` by patching the go templates for openapi-generator. I'm not sure we want to go down this road...